news_update.php

```diff
@@ -69,17 +69,15 @@
 
 echo '<div class="space-20"></div>';
 echo '<div class="alert alert-success">';
-echo lang_get( 'operation_successful' ) ?>
-
-echo '<br />';
+echo lang_get( 'operation_successful' );
+echo '<br /></div>';
 
 print_button( 'news_edit_page.php?news_id=' . $f_news_id . '&action=edit', lang_get( 'edit_link' ) );
+echo '&nbsp;&nbsp;&nbsp;&nbsp;';
```

Comment:
Bad news
> Ace admin template is retired and not for sale 
https://wrapbootstrap.com/theme/ace-responsive-admin-template-WB0B30DGR

@syncguru would it be good to have the non minified versions of the files to fix issues?
If so, they should be included in our source and we would have to add a step to our build process to create the minified versions.